cmd/agent/common/tagger_params.go

```diff
@@ -0,0 +1,40 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package common
+
+import (
+	"strings"
+
+	"github.com/DataDog/datadog-agent/comp/core/config"
+	tagger "github.com/DataDog/datadog-agent/comp/core/tagger/def"
+	"github.com/DataDog/datadog-agent/comp/core/tagger/types"
+	"github.com/DataDog/datadog-agent/pkg/api/security"
+	pkgconfigsetup "github.com/DataDog/datadog-agent/pkg/config/setup"
+	"github.com/DataDog/datadog-agent/pkg/config/utils"
+)
+
+// DualTaggerParams returns the params use inside the main agent
+func DualTaggerParams() (tagger.DualParams, tagger.Params, tagger.RemoteParams) {
```

Comment:
Could this be part of the tagger component ? Putting it in here mean it won't be usable by other processes.

Reply:
It would be usable for the agent subcommands. This is helpre function only used in the `cmd/agent/folder`. 

I think that putting as part of the components means that we are placing specific agent logic in the component package which I think is ideal. 

If you feel strongly about it I can move it, but I think here is a more accurate place to put the helper function